src/Core/ViewHelper/Traits/CompileWithContentArgumentAndRenderStatic.php

```diff
@@ -66,7 +75,7 @@ public function compile(
         ViewHelperNode $node,
         TemplateCompiler $compiler
     ) {
-        list ($initialization, $execution) = ViewHelperCompiler::getInstance()->compileWithCallToStaticMethod(
+        [$initialization, $execution] = ViewHelperCompiler::getInstance()->compileWithCallToStaticMethod(
```

Comment:
Since [Fluid still supports PHP >= 5.5](https://github.com/TYPO3/Fluid/blob/3d8499ae81e1fda39d236072f3ef8a7ef35433b1/composer.json#L6) we cannot do this.

Reply:
How came that ? This was a non intended change and not done activly oO. Will revert this.